### ISSUE TITLE
Run in the background and start automatically when the laptop starts

### ISSUE THREAD

**Issue by katzuv** [Galaxia5987/log-downloader#5]

**Comment by rakrakon** [comment]
Aren't we already doing this with Task Scheduler? @katzuv

**Comment by katzuv** [comment]
So I set up the issues using the new GitHub issues beta I sent you info about, so this is a sub-issue of the main issue. Look at #2, it has all the sub-issues organized in a tree there. 
I saw you addressed the requested changes, I'll review it in the morning.

**Comment by rakrakon** [comment]
Yeah i saw that, but why does this need an issue if it already works? Should i just close this one?

**Comment by katzuv** [comment]
It hasn't been tested yet and merged, so I wouldn't close
